docs/development/contributor_guidelines.rst

```diff
@@ -118,16 +118,33 @@ should be removed. If you are unfamiliar with how to squash commits with rebase,
 check out this
 `blog post <http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html>`__.
 
+.. _contributor-permissions:
+
 Privileges
 ----------
 
+.. note:: the translation workflow is different from the code and
+          documentation development workflow, see the associated
+          :ref:`privilege escalation <i18n-administrator-permissions>`
+          documentation.
+
 Dedicated contributors to SecureDrop will be granted extra privileges
-such as the right to push new branches or to merge pull
-requests. There is no formal process at the moment but the general
-idea is that any contributor with the right technical and social
-skills is entitled to ask. The people who have the power to grant such
-privileges are committed to do so in a transparent way to avoid any
-disputes.
+such as the right to push new branches or to merge pull requests. Any
+contributor with the right technical and social skills is entitled to
+ask. The people who have the power to grant such privileges are
+committed to do so in a transparent way as follows:
+
+* Step 1: the contributor posts a message `in the forum
+  <https://forum.securedrop.club/>`__ asking for privileges (review or
+  merge, etc.).
+* Step 2: after at least a week, someone with permissions to grant such
+  privilege reviews the thread and either:
+    * grants the privilege and adds a message to the thread
+    * explains what is expected from the contributor before they can be granted the privilege
+* Step 3: the thread is closed
+
+The privileges of a developer who has not been active for a year or
```

Comment:
what about 6 months? I feel like a year is a really long time (6 months is also what tor uses)

Reply:
absolutely ok, nice to have a point of reference like tor as well 👍

Reply:
done